samples/recognizer/from_file.go

```diff
@@ -60,18 +60,20 @@ func RecognizeOnceFromCompressedFile(subscription string, region string, file st
 	var containerFormat audio.AudioStreamContainerFormat
 	if strings.Contains(file, ".mulaw") {
 		containerFormat = audio.MULAW
-	}
+	} else
 	if strings.Contains(file, ".alaw") {
```

Comment:
For readibility I think would be better as follows.
if () {
} else if () {
} else if () {
} else {
}